### PR TITLE
different fixes towards making sure site updates & reads are done sequentially

### PR DESCRIPTION
**Pull request by dor-itzhaki**

### CODE REVIEW

**Comment by tomenden** [comment]
Related slack conversation - https://wix.slack.com/archives/CKD9ZUS8N/p1567600322006000

File: packages/corvid-local-site/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "scripts": {
     "build": ":",
-    "test": "jest"
+    "test": "jest --detectOpenHandles --forceExit"
```
Comment (tomenden) [inline packages/corvid-local-site/package.json]: This is an attempt to fix travis build?

Comment (dor-itzhaki) [reply packages/corvid-local-site/package.json]: yes

**Comment by tomenden** [comment]
@dor-itzhaki do you need help debugging breakage on travis? I can try to look at it tonight

**Pull request merged**